site/docs/webrtc/voice-iw.md

```diff
@@ -0,0 +1,191 @@
+---
+id: voice-iw
+title: Voice Interworking
+slug: /webrtc/voice-iw
+description: A description of available options for interworking with Bandwidth Programmable Voice
+keywords:
+  - bandwidth
+  - webrtc
+  - video
+image: ../../static/img/bandwidth-logo.png
+---
+
+## Connecting WebRTC to Programmable Voice
+
+Bandwidth has a demonstrated history of excellence in IP voice technologies, including comprehensive CPaaS Programmable Voice and Messaging capabilities. The introduction of WebRTC capabilities provides the opportunity to combine the strength and diversity of our Programmable Voice capabilities with our highly controllable WebRTC capabilities to yield a rich and flexible platform that allows our customers to extend their business with a broad range of communication capabilities.
```

Comment:
See below.... IMO we're using the word 'capabilities' too much:

Bandwidth has a demonstrated history of excellence in IP voice technologies, including comprehensive CPaaS Programmable Voice and Messaging. The introduction of WebRTC provides the opportunity to combine the strength and diversity of our Programmable Voice capabilities with WebRTC to yield a rich and flexible platform that allows our customers to extend their business with a broad range of communication capabilities.

Reply:
I agree - I got rid of 3 instances - thanks